### PR TITLE
Log

### PR DESCRIPTION
**Pull request by luis9614**
Includes
* log functionality
* PEP8
* Input verification for 25 digits and 10 decimals

### CODE REVIEW

File: models/function.py [hunk 1]
```diff
@@ -6,4 +11,29 @@ def __init__(self, index, name, func, symbol):
         self.symbol = symbol
 
     def execute(self, a, b):
-        return self.func(a, b)
\ No newline at end of file
+        result, success = self.func(a, b)
+        self.save_to_log(a, b, result, success)
+        return result, success
+
+    def save_to_log(self, a, b, result, success):
```
Comment (dach19) [inline models/function.py]: agrega docstring

File: models/function.py [hunk 2]
```diff
@@ -6,4 +11,29 @@ def __init__(self, index, name, func, symbol):
...
+        date_str = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+        if success:
+            log_msg = "%s\t %s %s %s = %f" % (date_str, a,
```
Comment (dach19) [inline models/function.py]: Usa .format()

File: output/log.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+2019-11-06 10:14:01	 Could not compute operation: division (3.4.5 / 34)
```
Comment (dach19) [inline output/log.txt]: también agrega este archivo a .gitignore.

File: utils/utils.py [hunk 4]
```diff
@@ -1,6 +1,31 @@
-def getMenu(operations):
+from models.exceptions import InvalidNumberFormat
+from models.errors import ProgramErrors
+
+def get_menu(operations):
     menu_text = ""
     for opt in operations:
-        menu_text = menu_text + "\n" + str(operations[opt].index) + " " + operations[opt].name
+        menu_text = menu_text + "\n" + str(operations[opt].index) \
```
Comment (dach19) [inline utils/utils.py]: usa .format()

File: utils/utils.py [hunk 5]
```diff
@@ -1,6 +1,31 @@
-def getMenu(operations):
+from models.exceptions import InvalidNumberFormat
+from models.errors import ProgramErrors
+
+def get_menu(operations):
```
Comment (dach19) [inline utils/utils.py]: agrega docstring.

File: utils/utils.py [hunk 6]
```diff
@@ -1,6 +1,31 @@
...
-        menu_text = menu_text + "\n" + str(operations[opt].index) + " " + operations[opt].name
+        menu_text = menu_text + "\n" + str(operations[opt].index) \
+                    + " " + operations[opt].name
     menu_text += "\n\n"
-    return "\nselect an operation" + menu_text + "\npress 'q' to quit\n\n"
\ No newline at end of file
+    return "\nselect an operation" + menu_text + "\npress 'q' to quit\n\n"
+
+
+def verify_input_length(ni):
```
Comment (dach19) [inline utils/utils.py]: agrega docstring.

Comment (luis9614) [reply utils/utils.py]: Fixed on cac768c5266f56e43e7d03306e6bbde67239cb45

**Comment by luis9614** [comment]
All requested changes have been fixed on cac768c5266f56e43e7d03306e6bbde67239cb45

**Pull request merged**